src/main/java/CheckoutTest.java

```diff
@@ -0,0 +1,57 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.Select;
+
+public class CheckoutTest extends BaseTest{
+    @Before
```

Comment:
-in @Befores de regula nu facem verificari, tu ai asserturi si dai impresia ca le consideri metode de test
-nu are rost sa ai ai doua @Befores, si daca ai doua, nu se vor reflecta in raportare, iar de rulat vor rula la fel ca si intr-una singura